spec/models/book_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe Book, type: :model do
+  context 'validity of Book' do
+    let(:book1) do
+      Book.create(
+        author_name: author_name,
+        title: title,
+        description: description,
+        rating: rating,
+        word_count: word_count
+      )
+    end
+    let(:author_name) { 'Annie Walsh' }
+    let(:title) { 'A memoir' }
+    let(:description) { 'The amazing life story' }
+    let(:word_count) { 100_000 }
+    let(:rating) { 5 }
+
+    let(:book2) do
+      Book.create(
+        author_name: author_name,
+        title: title,
+        description: description,
+        rating: rating,
+        word_count: word_count
+      )
+    end
+    let(:author_name) { 'Kathleen Walsh' }
+    let(:title) { 'A memoir' }
+    let(:description) { 'The new and improved life story' }
+    let(:word_count) { 250_000 }
+    let(:rating) { 4 }
+
+    it 'is valid with valid attributes' do
+      expect(book1).to be_valid
+    end
+
+    it 'is not valid without a unique title' do
+      book1
```

Comment:
This is valid.

It's also an opportunity to introduce two new RSpec concepts.

**1. Before and After**

```ruby
# Our SomeClass requires the name to be unique
RSpec.describe SomeClass do
  let(:instance_1) { SomeClass.create(name: 'a') }
  let(:instance_2) { SomeClass.create(name: 'a') }

  # a before block is used to setup things needed before a test is run.
  # They increase readability of tests.
  before { instance_1 } # this will instantiate the variable

  # P.S. Take a look at before. It's just Ruby. It's an instance method that accepts no args and a block (like all Ruby methods accept a block). Then Rspec ensures it's called before each `it` in a scope.
  # Gems can often seem like another language (and with their DSL they almost are from one perspective). But in truth it's Ruby all the way down.
  
  it 'must have unique title' do
    expect(instance_2).to_not be_valid
  end

  # you can also use after blocks to do stuff after. Same idea, otherside. I rarely (never??) need these.
  after {}
end
```

**let!()**
  
  Our example above is simple. The `before` block contains one statement, to instantiate the instance of the class.

This use is common enough that RSpec provides a shorthand.

```ruby
# Our SomeClass requires the name to be unique
RSpec.describe SomeClass do
  # the ! bangs it up.
  # Which is to say, is ensures the variable is instantiated (the statement is evaluated) before every test in the current scope.
  # instance_1 will always be made.
  let!(:instance_1) { SomeClass.create(name: 'a') }
  let(:instance_2) { SomeClass.create(name: 'a') }
  
  it 'must have unique title' do
    expect(instance_2).to_not be_valid
  end
end
```

Play around with these, pick one to use (sticking with your current approach is one of the options).